tests/unit/simple_tests.js

```diff
@@ -146,6 +159,25 @@ Object.keys(env).forEach(key => {
                 });
         });
 
+        it('should get Bucket informations on an existing bucket', done => {
+            client.getBucketInformation(existBucket.name, reqUids,
+                (err, data) => {
+                    const ret = JSON.parse(data);
+                    assert.deepStrictEqual(ret, existBucket.bucketInformation);
+                    done(err);
+                });
+        });
+
+        it('should get Bucket informations on an unexisting bucket', done => {
+            client.getRaftInformation(nonExistBucket.name, reqUids,
+                err => {
+                    const error = errors.NoSuchBucket;
```

Comment:
We should create a new object in order not to modify the original error from arsenal `errors`.